### ISSUE TITLE
[BUG]: Release 0.37.0 not compatible with community edition?

### ISSUE THREAD

**Issue by ja-p** [activepieces/activepieces#6167]
**Describe the bug**
I self host Activepieces (Community) using docker and when I upgraded to 0.37.0, the container would not start anymore due to the following error:

```
Migration "CreateProjectRoleTable1731424289830" failed, error: relation "project_member" does not exist
{"level":50,"time":1732759981986,"pid":8,"hostname":"cbc29c504d44","err":{"type":"QueryFailedError","message":"relation \"project_member\" does not exist","stack":"QueryFailedError: relation \"project_member\" does not exist\n    at PostgresQueryRunner.query (/usr/src/app/dist/packages/server/api/node_modules/typeorm/driver/src/driver/postgres/PostgresQueryRunner.ts:331:19)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)","query":"\n            ALTER TABLE \"project_member\" ADD COLUMN \"projectRoleId\" character varying\n        ","driverError":{"type":"DatabaseError","message":"relation \"project_member\" does not exist","stack":"error: relation \"project_member\" does not exist\n    at /usr/src/app/dist/packages/server/api/node_modules/pg/lib/client.js:526:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at PostgresQueryRunner.query (/usr/src/app/dist/packages/server/api/node_modules/typeorm/driver/src/driver/postgres/PostgresQueryRunner.ts:260:25)","length":112,"name":"error","severity":"ERROR","code":"42P01","file":"namespace.c","line":"433","routine":"RangeVarGetRelidExtended"},"length":112,"severity":"ERROR","code":"42P01","file":"namespace.c","line":"433","routine":"RangeVarGetRelidExtended"},"msg":"[Main#main]"}
```

**To Reproduce**
1. Upgrade docker image to latest (0.37.0) while using community edition
2. Attempt to start container.
3. See error

**Expected behavior**
Migrations should execute and the server should start running

**Screenshots**
None.

**Additional context**
I did quick troubleshooting and I think I got the root cause, but I welcome you to verify this. I believe this is due to CreateProjectRoleTable1731424289830 being a common migration and it attempts to alter the table "project_member"
```
            ALTER TABLE "project_member" ADD COLUMN "projectRoleId" character varying
```
This table only gets created if the edition is at least Enterprise with the following migration: AddProjectMembers1689177797092.

Could that be it, or is something wrong with my setup?

**Comment by kerta1n** [comment]
+1 for this issue, running Community on Unraid using the IBRACORP template

**Comment by Creogenesis** [comment]
+1 same error

**Comment by abuaboud** [comment]
I am on it!

**Comment by abuaboud** [comment]
Sorry for this, It's now fixed in 0.37.1

**Comment by kerta1n** [comment]
For the time being, Unraid users may need to manually specify version instead of `latest` to get around this issue
